.github/workflows/build-release.yml

```diff
@@ -0,0 +1,41 @@
+## This is triggered by the publishing of a release, and will build the assets and attach them.
+
+name: On Release 
+
+on:
+  release:
+    types: [published]
+        
+jobs:
+  ci:
+    name: CI
+    uses: ./.github/workflows/ci.yml
+    with:
+      debug_build: false
+
+  release:
+    name: Attach Release Artifacts
+    needs: ci
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Code Checkout
+        uses: actions/checkout@v2
```

Comment:
If we continue with the automatic execution, maybe we can, to be sure that we download what we want, specify the tag to be sure that the checkout is the correct one?

I've seen that this action has a version 3. Maybe is good too start to use the latest version.

Reply:
Is it not automatically the commit that triggered the run of the job?